packages/components/src/Search.purs

```diff
@@ -63,16 +65,24 @@ type TaggedSubscription =
   , expired :: Boolean
   }
 
+type EmailPassword =
+  { email :: String
+  , password :: Maybe String
+  }
+
+data PersonaUserEdit
+  = SetCusno (AsyncWrapper.Progress (Maybe Cusno))
+  | ControlPassword (AsyncWrapper.Progress EmailPassword)
+
 search :: Component Props
 search = do
   component "Search" \ { setActiveUser, resetRedirect, now } -> React.do
-    query /\ setQuery <- useState' Nothing
+    query /\ setQuery <- useState' $ Just "kari.pahula+demo5@ksfmedia.fi"
```

Comment:
Is your email here on purpose? 🙂

Reply:
Oops. I got tired of typing it after nth iteration.